<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <!-- decawave_node.launch
       VERSION 0.0.0 Last Changed 2019-04-01
  -->
  <!-- Interface:
    Pub: odometry/gps (nav_msgs/Odometry)
      The position estimate of the decawave, with variances.
    Param: frequency (double) 50.0; The frequency of the read loop
  -->
  <node pkg="decawave" type="decawave_node" name="decawave_node" output="screen" respawn="true">
    <param name="frequency" value="50.0"/>
    <param name="port_name" value="ttyACM0"/>
  </node>
</launch>
